docs/linter-usage.md

```diff
@@ -5,10 +5,10 @@
 In order to get reports for all files in repository, run the following:
 
 ```sh
-$ noverify -cache-dir=$HOME/tmp/cache/noverify /path/to/your/project/root
+$ noverify check [-cache-dir=$HOME/tmp/cache/noverify] /path/to/your/project/root
```

Comment:
I don't think that `[` and `]` are intuitive.
Usually, these snippets are interpreted as something you can copy/paste and use right away.
You can make it 2 examples: 1 with default cache and another with a custom location.
Comments can help to make it obvious why you may need to use a custom cache location and what's the behavior of the default one.